server/routes/vas.routes.js

```diff
@@ -0,0 +1,126 @@
+import express from 'express';
+import { addVASSchema, updateVASSchema } from '../schemas/vas.schema';
+import { schemaValidation, authenticate, isAdmin, hasValidObjectId } from '../helpers/middleware';
+import VASController from '../controllers/vas.controllers';
+
+const router = express.Router();
+
+/**
+ * @swagger
+ * /vas/:
+ *   post:
+ *     tags:
+ *       - VAS
+ *     description: Allows an admin add a value added service
+ *     produces:
+ *       - application/json
+ *     requestBody:
+ *      content:
+ *        application/json:
+ *          schema:
+ *            $ref: '#/components/schemas/VAS'
+ *      description: Allows an admin add a value added service
+ *     responses:
+ *      '201':
+ *        description: Value added service added successfully
+ *      '400':
+ *        description: Error adding value added service
+ *      '500':
+ *       description: Internal server error
+ */
+router.post('/', authenticate, isAdmin, schemaValidation(addVASSchema), VASController.addVas);
+
+/**
+ * @swagger
+ * /vas/:
+ *   put:
+ *     tags:
+ *       - VAS
+ *     description: Updates existing Value added service
+ *     produces:
+ *       - application/json
+ *     requestBody:
+ *      content:
+ *        application/json:
+ *          schema:
+ *            $ref: '#/components/schemas/VAS'
+ *      description: Updates existing Value added service
+ *     responses:
+ *      '200':
+ *        description: VAS updated
+ *      '400':
+ *        description: Error updating VAS
+ *      '500':
+ *       description: Internal server error
+ */
+router.put('/', authenticate, isAdmin, schemaValidation(updateVASSchema), VASController.updateVas);
```

Comment:
For javascript variables, methods, models, and services can we write VAS in normal letter conventions to ensure consistency. I see lots of `vas`, `Vas`, and `VAS`  in different places for different things in the code. 

To maintain coding consistency and prevent unnecessary typos in the future, can we just try VAS like normal text like User, or Vendor and ensure we follow the proper javascript naming convention for names that involve VAS.

```suggestion
router.put('/', authenticate, isAdmin, schemaValidation(updateVasSchema), VasController.updateVas);
```